loopd/swapclient_server.go

```diff
@@ -617,6 +651,11 @@ func filterSwap(swapInfo *loop.SwapInfo, filter *looprpc.ListSwapsFilter) bool {
 		return false
 	}
 
+	// If timestamp filters are set, only return swaps within the specified time range.
+	if filter.StartTimestampNs > 0 && swapInfo.InitiationTime.UnixNano() < filter.StartTimestampNs {
```

Comment:
Also please format this line to avoid too long line:

```go
if filter.StartTimestampNs > 0 &&
  swapInfo.InitiationTime.UnixNano() < filter.StartTimestampNs {

  return false
}
```

We use 80 as a limit, counting tab as 8.

Reply:
Thanks again